### PR TITLE
Stl 1521 maybepage

### PR DESCRIPTION
**Pull request by SannyNguyenHung**
# Short Description
- Changed texts for maybe page & add detail for unlock code. This is the frontend part of STL-1521 - but the changes from the backend are integrated here. 

# Changes
- Insert component into the maybe page
- Changed some texts

# Files
* webapp/app/templates/eligibility/display_maybe.html 
* webapp/app/forms/steps/eligibility_steps.py

# Feedback
- Is it ok how I have integrated the component render_detail into the maybe page? Have not seen this anywhere else.

### CODE REVIEW

**Review by frederike-ramin — CHANGES_REQUESTED** [review]
Nice work with the details, I think the dependent notes aren't quite correct yet though.

File: webapp/app/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -5380,3 +5318,89 @@ msgstr ""
...
+
+#~ msgid "form.eligibility.elster_abrufcode_eligibility.yes"
+#~ msgstr "Ja"
+
+#~ msgid "form.eligibility.elster_abrufcode_eligibility.no"
+#~ msgstr "Nein"
+
+#~ msgid "form.eligibility.elster_abrufcode_eligibility.unknown"
+#~ msgstr "Weiß ich nicht"
+
```
[the remark this answers is not in the document]
Comment (SannyNguyenHung) [reply webapp/app/translations/de/LC_MESSAGES/messages.po:5406]: Removed it :) thanks! :+1:

File: webapp/app/forms/steps/eligibility_steps.py [hunk 2]
```diff
@@ -762,8 +762,20 @@ class EligibilityMaybeDisplaySteuerlotseStep(EligibilityStepMixin, DisplaySteuer
...
         super(EligibilityMaybeDisplaySteuerlotseStep, self).__init__(endpoint=endpoint,
                                                                      stored_data=stored_data,
                                                                      header_title=_('form.eligibility.header-title'),
-                                                                     *args, **kwargs)
+                                                                     *args, **kwargs)        
+        
+        
+    def _main_handle(self):  
+        super()._main_handle()
+        dependent_notes = [(_('form.eligibility.result-note.capital_investment'))]
```
[the remark this answers is not in the document]
Comment (SannyNguyenHung) [reply webapp/app/forms/steps/eligibility_steps.py:774]: Hehe :smile: your right... Calling a function here now for both pages

File: webapp/app/templates/eligibility/display_maybe.html [hunk 3]
```diff
@@ -21,6 +21,10 @@ <h1 class="headline">{{ render_info.step_title }}</h1>
         </div>
     </div>
 
+    <div class="spacing-b-09">
+        {{ components.render_detail(render_info.detail) }}
+    </div>
```
[the remark this answers is not in the document]
Comment (SannyNguyenHung) [reply webapp/app/templates/eligibility/display_maybe.html:26]: Done!

File: webapp/app/forms/steps/eligibility_steps.py [hunk 4]
```diff
@@ -762,20 +751,33 @@ class EligibilityMaybeDisplaySteuerlotseStep(EligibilityStepMixin, DisplaySteuer
...
               
-        self.render_info.additional_info['dependent_notes'] = dependent_notes
+        self.render_info.additional_info['dependent_notes'] = get_dependent_note(self)
         self.render_info.detail = {'render_kw': {
                 'data-detail': {'title': _l('form.eligibility.result-note.when_unlock_code.title'),
                                 'text': _l('form.eligibility.result-note.when_unlock_code.description')}}
                 }  
+
+
+def get_dependent_note(self):
```
Comment (frederike-ramin) [inline webapp/app/forms/steps/eligibility_steps.py:772]: Can we name this get_dependent_notes as it returns a list?

**Review by frederike-ramin — APPROVED** [review]
Nice. Just one very nit-picky comment about naming, otherwise it looks good. 👍 
Just make sure to first merge #356.

File: webapp/app/forms/steps/eligibility_steps.py (hunk 4 above)
Comment (SannyNguyenHung) [reply webapp/app/forms/steps/eligibility_steps.py:772]: Done!

**Pull request merged**